helm/dagger/values.yaml

```diff
@@ -0,0 +1,50 @@
+nameOverride: ""
+fullnameOverride: ""
+
+engine:
+  # Set engine.config if you needa custom Dagger Engine config file
+  config: |
+    debug = true
+    insecure-entitlements = ["security.insecure"]
+    [registry."registry:5000"]
+      http = true
+    
+    [registry."docker.io"]
+      mirrors = ["mirror.gcr.io"]
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    ##
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    ##
+    requests: {}
```

Comment:
We should add a comment here about matching `--oci-max-parallelism` to this value. Considering how this flag behaves, I cannot imagine a scenario where I would recommend allocating less than `cpu: 1` for either `requests` or `limits`.

I also don't think than anything less than `memory: 1Gi` will give users a good experience by default.

Reply:
I don't think we should match `--oci-max-parallelism` with this value in the general case.

One of the ways that we (and Airbyte) are using the DaemonSet is as a "noisy neighbor" that utilizes CPU and Memory resources that are requested by the GHA jobs. It helps create a correctly sized node with Karpenter.